Deploy-CAM.ps1

```diff
@@ -453,8 +453,8 @@ function New-RemoteWorstationTemplates
 	$DomainAdminUsername = $CAMConfig.ARMParameters.parameters.domainAdminUsername.value
 	$djSecretName = $CAMConfig.internal.djSecretName
 	$rcSecretName = $CAMConfig.internal.rcSecretName
-	$rwLaSecretName = $CAMConfig.internal.rwLaSecretName
-	$csLaSecretName = $CAMConfig.internal.csLaSecretName
+	$rwLocalSecretName = $CAMConfig.internal.rwLocalSecretName
+	$csLaSecretName = $CAMConfig.internal.csLocalSecretName
```

Comment:
Is this variable referenced anywhere in this function? Seems maybe not ($csLaSecretName)